### PR TITLE
Add comment for intentional duplicated monospace font

### PR DESCRIPTION
**Pull request by fritx**
https://github.com/necolas/normalize.css/issues/480
https://github.com/necolas/normalize.css#pre-code-kbd-samp

I was confused until I found the answer, so it's better to add this missing comment ;)

### CODE REVIEW

**Comment by 7studio** [comment]
:+1:

**Comment by KIPdeKIP** [comment]
This should definitely be added. It's pretty obvious that there's confusion why it's there with it being asked at least 12 times now in: #196, #293, #295, #326, #328, #335, #346, #347, #374, #415, #476 and #480.

**Comment by fritx** [comment]
wow~

**Comment by battaglr** [comment]
It will certainly reduce the number of issues opened by this. :+1:

File: normalize.css [hunk 1]
```diff
@@ -226,11 +226,15 @@ pre {
  * Address odd `em`-unit font size rendering in all browsers.
  */
 
+/**
+ * 1. Fix the inheritance and scaling of font-size for preformated text.
```
Comment (cvrebert) [inline normalize.css]: It's spelled "preformatted" with two Ts.

Comment (fritx) [inline normalize.css]: Damn. Just had it copied from here: https://github.com/necolas/normalize.css/blob/2bdda84272650aedfb45d8abe11a6d177933a803/README.md#pre-code-kbd-samp

**Comment by jonathantneal** [comment]
@fritx, great work. Merged in 5daad01! @KIPdeKIP, @battaglr, there go all the easy issues to close. :stuck_out_tongue_closed_eyes:

**Comment by battaglr** [comment]
@jonathantneal: awesome! If I can be of any help with the remaining issues, just let me know!

**Comment by jonathantneal** [comment]
Thanks, @battaglr! I’m trying to resolve every PR from oldest to newest. If you want to tackle any one of them, let me know. Then we move onto issues.

**Comment by fritx** [comment]
Awesome!

**Comment by HelderMagalhaes** [comment]
Just bumped into this. Seeing duplicated default classes in optimized CSS files is unexpected, so I ended up here on investigating. A straight `duplication is intentional` comment next to `monospace, monospace` occurrences was better IMO to avoid confused users and erroneous error reports. Should an additional issue be created or could this still "fit" this 5yo one?

**Pull request closed without merging**